api/scenario_api_utils/package.xml

```diff
@@ -15,23 +15,20 @@
   <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
   <license>TODO</license>
```

Comment:
@mitsudome-r can you choose a license here?

Reply:
Done.